src/Entities/CollectionQuery.ts

```diff
@@ -8,6 +8,10 @@ export class CollectionQuery {
   @ObjectIdColumn()
   id: ObjectID
 
+  @Field(type => ID, { nullable: true })
+  @ObjectIdColumn({ name: "id" })
+  internalID: ObjectID
```

Comment:
Same question here. I get that it follows the declaration above, but assuming that these are DB IDs, I don’t see why they can be nullable.